docs/server_notices.md

```diff
@@ -61,14 +61,33 @@ displayname and avatar of the Server Notices user.
 Sending notices
 ---------------
 
-As of the current version of synapse, there is no convenient interface for
-sending notices (other than the automated ones sent as part of consent
-tracking).
+The API to send notices is as follows:
 
-In the meantime, it is possible to test this feature using the manhole. Having
-gone into the manhole as described in [manhole.md](manhole.md), a notice can be
-sent with something like:
+```
+POST /_matrix/client/r0/admin/send_server_notice/[<user_id>]
```

Comment:
so is it r0 or v1?

Reply:
it was a copy/paste error. I got fooled by the wrong doc strings of the other methods.
Fixed now